src/polls/PollForm.vue

```diff
@@ -244,26 +402,28 @@ export default defineComponent({
     },
   },
   mounted() {
-    this.$store.dispatch(`${StoreModuleEnum.housesStore}/${HousesActionsEnum.SET_HOUSES}`, this.$props.cooperationId);
+    if (this.$props.isEditing) {
+      try {
+        this.$store.dispatch(`${StoreModuleEnum.housesStore}/${HousesActionsEnum.SET_HOUSES}`, this.cooperationIdEdit);
+        this.initData();
+      } catch {
+        console.log('error was caught during mounting BaseCooperationPoll');
+      }
+    } else if (!this.$props.isEditing) {
```

Comment:
Do you need this condition in 'else if' or just 'else' will do?

Reply:
I agree with you, "else" is enough here and it is working pretty well
I just wanted to single out and clearly indicate the type so that there was no unpredictable option. But as we are using this logic only in polls, therefore there is no other situation and no need in "else if" construction